File: src/programmers_prac/lessons12933.java
package programmers_prac;
//연습문제
//정수 내림차순으로 배치하기

import java.util.Arrays;

public class lessons12933 {

    public long solution(long n) {
        String tmp = "";
        char[] charArr = (n + "").toCharArray();
        Arrays.sort(charArr);
        for (int i = charArr.length - 1; i >= 0; i--) {
            tmp += charArr[i];
        }
        return Long.parseLong(tmp);
    }

}

File: src/programmers_prac/lessons12933.js
function solution(n) {
    var answer = "";
    let a = Array.from(n.toString());
    a.sort((o1, o2) => {
        return o2 - o1;
    })
    a.forEach((e)=>{
        answer += e;
    })
    return Number(answer);
}

solution(118372);